### PR TITLE
Make Pulumi pass the correct migrationId for Validator-stable-old 

### PR DESCRIPTION
**Pull request by pasindutennage-da**
Fix https://github.com/DACH-NY/canton-network-internal/issues/6411

### CODE REVIEW

File: token-standard/package-lock.json [hunk 1]
```diff
@@ -42,23 +42,23 @@
     },
     "cli/daml.js/daml-stdlib-DA-Time-Types-1.0.0": {
```
Comment (pasindutennage-da) [inline token-standard/package-lock.json:43]: did a make clean-all && make build

**Review by moritzkiefer-da — CHANGES_REQUESTED** [review]
please add an integration test that configures the sv and scan app with a non-zero migration id and check that a traffic purchase works (and confirm that it fails without this)

**Comment by pasindutennage-da** [comment]
@moritzkiefer-da 

> please add an integration test that configures the sv and scan app with a non-zero migration id and check that a traffic purchase works

`SV1Initializer`:  `domainMigrationId <- resolveDomainMigrationId(Future.successful(0L))`

Hence asserting against scan.getMigrationId() just evaluates to 0 == 0. This won't catch any future regressions.

I don't see an exposed config parameter to force a non-zero migration ID on boot. Any tips? 

Should I do some expensive `upgrade` first (which sounds a bit out of scope for MemberTraffic test)?

**Comment by moritzkiefer-da** [comment]
> Hence asserting against scan.getMigrationId() just evaluates to 0 == 0. This won't catch any future regressions.

maybe just add a config flag to overwrite in testing

File: apps/validator/src/main/scala/org/lfdecentralizedtrust/splice/validator/automation/TopupMemberTrafficTrigger.scala [hunk 2]
```diff
@@ -59,7 +59,6 @@ class TopupMemberTrafficTrigger(
     clock: Clock,
     walletManager: UserWalletManager,
     scanConnection: BftScanConnection,
-    domainMigrationId: Long,
```
Comment (nicu-da) [inline apps/validator/src/main/scala/org/lfdecentralizedtrust/splice/validator/automation/TopupMemberTrafficTrigger.scala:62]: why is this needed though? shouldn't the trigger be initialized with the same migration id that's resolved from scan?

**Review by nicu-da — CHANGES_REQUESTED** [review]
I don't think we're solving anything here, the id is read from the db or from scan, there's no configurable migration id anymore.

The issue is that we updated the migration id for a version that still had the migration id in the config, so we should just fix that deployment

File: apps/validator/src/main/scala/org/lfdecentralizedtrust/splice/validator/automation/TopupMemberTrafficTrigger.scala (hunk 2 above)
Comment (pasindutennage-da) [reply apps/validator/src/main/scala/org/lfdecentralizedtrust/splice/validator/automation/TopupMemberTrafficTrigger.scala:62]: @nicu-da 

Trigger should be initialized with the resolved value, however, trigger should dynamically fetch it from scan in future (otherwise, it will keep using a stale id.) Thats the central idea of this PR, and the attached issue details the reasoning.

Comment (nicu-da) [reply apps/validator/src/main/scala/org/lfdecentralizedtrust/splice/validator/automation/TopupMemberTrafficTrigger.scala:62]: @pasindutennage-da the migration id is static and will never change anymore, what's the situation where we see it changing?

**Comment by moritzkiefer-da** [comment]
> The issue is that we updated the migration id for a version that still had the migration id in the config, so we should just fix that deployment

The problem we saw was that new validators on cilr submitted traffic purchases with migration id 0. Definitely possible I misdiagnosed but it is a very bad issue so we definitely need to do something™.

**Review by nicu-da — COMMENTED** [review]
as discussed in dm, lets just add an additional_config to the validator app in cilr for the old validator
